### DISCUSSION TITLE
Metabolism and disposition parameters for PBPK model

### DISCUSSION THREAD

**Discussion by AjinkyaJ11** [Open-Systems-Pharmacology/Forum#1243] [Models]
We are working on PBPK modelling for a highly soluble BCS class III molecule. The formulation is an immediate release molecule with complete release in 15 min(in-vitro). The Tmax for this molecule is 13 -15 hr with secondary peak at 30 -35 hrs(maybe due to invivo phosphorylation/oxidation). what should be metabolism and disposition to build physiology for such molecule? With in-build physiology and literature clearance parameters?
 Tmax is 2 hrs. Below is the plasma concentration profile. 
Second point need to address is second peak. If second peak is due to reabsorption. Then how this should  be built in physiology. 
![image](https://user-images.githubusercontent.com/108856607/177702405-cb00bbe2-e060-41a3-aa6e-3d6cfb15a5e7.png)

**Comment by StephanSchaller** [comment]
This is a quite non-trivial question with a lot of different aspects to it and requires a bit more time to answer.
My first quick feedback would be to think about an enterohepatic cycle with conjugation (liver) and deconjugation (large intestine) of your compound and reabsorption in the large intestine (similar to bile-acid re-circulation)... can be done in MoBi after some preparations in PK-Sim.
